docs/source/docs/level-up/intermediate_skills/09_data_filtering.rst

```diff
@@ -0,0 +1,56 @@
+===========================
+Level 9: Dataset Filtering
+===========================
+
+With the increasing availability of public data, the need for data filtering has become more apparent. Raw data often
+contains irrelevant or unnecessary information, making it difficult to extract the desired insights or use it effectively
+for decision-making purposes. Data filtering involves the process of identifying and selecting relevant data points while
+excluding or removing the irrelevant ones to improve the quality and usability of the data. This process is essential for
+ensuring that data can be used effectively and efficiently to drive insights and inform decisions. As the volume and complexity
+of data continue to grow, data filtering will become an increasingly important aspect of data management and analysis.
+By filtering the dataset in this way, we can create a subset of data that is tailored to our specific needs, making it easier
+to extract meaningful inshights or use it effectively for decision-making purposes.
+
+In this tutorial, we provide the simple example of filtering dataset using item and annotation. The detailed deescription for filter
+operation is given by :doc:`Filter <../../command-reference/context_free/filter>`. The more advanced Python example is given
+:doc:`this notebook <../../jupyter_notebook_examples/notebooks/04_filter>`.
+
+.. tab-set::
+
+    .. tab-item:: ProjectCLI
+
+        With the project-based CLI, we first create project and import datasets into the project
+
+        .. code-block:: bash
+
+            datum project create --output-dir <path/to/project>
+            datum project import --format datumaro --project <path/to/project> <path/to/data>
+
+        We filter dataset through
+
+        .. code-block:: bash
+
+            datum filter -e <how/to/filter/dataset> --project <path/to/project>
```

Comment:
Can you give a small example not just giving `<how/to/filter/dataset>`?

Reply:
Fix in [`1104c4d`](https://github.com/openvinotoolkit/datumaro/commit/1104c4d3e8e39592e6e7993aa2ab494b45acea64). I added some example for how to set filter in each tab.